mlir/lib/Dialect/MIOpen/Transforms/AffineTransforms.cpp

```diff
@@ -56,6 +59,15 @@ AffineMap AffineTransforms::buildIndexAffineMap(miopen::TransformOp op) {
       } else if (transformAttr.getValue() == "Pad") {
         assert(srcDimAttr.size() == destDimAttr.size());
 
+        auto limitDimAttr = dimLayoutAttr.get("bound_check");
```

Comment:
maybe bound_check_read & bound_check_write is better

Reply:
From the current demand, no need. We can add when need